### PR TITLE
Added support for Livarnolux (Lidl) remote control

### PR DESCRIPTION
**Pull request by KubaDee**
Adding new device - Livarnolux (Lidl) remote control

Device photo:
https://www.lidl-shop.cz/media/9fd41c55fb300050cdb2dc54f19ecd71.jpeg

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -15012,6 +15012,25 @@ const devices = [
...
+        model: 'TS1001',
+        vendor: 'Livarnolux',
+        description: 'Livarnolux remote control dimmer',
+        supports: 'switch and dimming',
+        exposes: [e.action([
+            'on', 'off', 'brightness_down_click', 'brightness_up_click', 'brightness_down_hold', 'brightness_up_hold',
+            'brightness_down_release', 'brightness_up_release',
+        ])],
+        fromZigbee: [
+            fz.command_on, fz.legacy_genOnOff_cmdOn, fz.command_off, fz.legacy_genOnOff_cmdOff, fz.CTR_U_brightness_updown_click,
```
Comment (Koenkk) [inline devices.js:15028]: Please only use the `fz.comand_*` converters here, the rest is all legacy, e.g. `CTR_U_brightness_updown_click` -> `command_step`.

File: devices.js [hunk 2]
```diff
@@ -15012,6 +15012,25 @@ const devices = [
             return {'l1': 1, 'l2': 2, 'l3': 3};
         },
     },
+
+    // Livarnolux (Lidl) remote control dimmer
+    {
+        zigbeeModel: ['TS1001'],
+        model: 'TS1001',
+        vendor: 'Livarnolux',
```
Comment (Koenkk) [inline devices.js:15020]: I expect the vendor to be Silvercrest

File: devices.js [hunk 3]
```diff
@@ -15012,6 +15012,25 @@ const devices = [
...
+        description: 'Livarnolux remote control dimmer',
+        supports: 'switch and dimming',
```
Comment (Koenkk) [inline devices.js:15022]: `supports` can be removed (not used anymore)

File: devices.js [hunk 4]
```diff
@@ -15012,6 +15012,25 @@ const devices = [
             return {'l1': 1, 'l2': 2, 'l3': 3};
         },
     },
+
+    // Livarnolux (Lidl) remote control dimmer
+    {
+        zigbeeModel: ['TS1001'],
+        model: 'TS1001',
```
Comment (Koenkk) [inline devices.js:15019]: Fill in the real model here (probably written on the packaging), I expect it to start with HG

**Comment by Koenkk** [comment]
Looks good, as last, can you provide me the data/database.db entry of this device?

**Comment by KubaDee** [comment]
Here is a record from  DB.

`{"id":10,"type":"EndDevice","ieeeAddr":"0xec............","nwkAddr":32975,"manufId":4098,"manufName":"_TYZB01_bngwdjsr","powerSource":"Battery","modelId":"TS1001","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":260,"inClusterList":[0,1,3,4,4096],"outClusterList":[25,10,3,4,5,6,8,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TS1001","manufacturerName":"_TYZB01_bngwdjsr","powerSource":3,"zclVersion":3,"appVersion":81,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x0012..........","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0x0012.....","endpointID":1}],"meta":{}}},"appVersion":81,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1606482945053}`

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**